fastlane/lanes/build.rb

```diff
@@ -329,6 +329,11 @@ def upload_installable_build(product:)
       reuse_identifier: "#{product.downcase}-installable-build-link",
       body: comment_body
     )
+
+    if ENV['BUILDKITE']
```

Comment:
Nitpick: Having worked with RuboCop a bunch recently, I think it would say something like "use `fetch` please".

I'm not 100% sure of the syntax so no `suggestion` block, and it doesn't really matter in this context IMHO. It should be something like

```ruby
if ENV.fetch('BUILDKITE', false) # maybe `false` is unnecessary as `fetch` will return `nil` if not found?
```

Reply:
FYI: I borrowed that env var name and the syntax to check it… from [a couple of lines just below that](https://github.com/wordpress-mobile/WordPress-Android/blob/b4c4a916f4a4276dfe6564c88455e5d8d5150cea/fastlane/lanes/build.rb#L341-L344). So if we change it here at some point to make rubocop happy or use a better style, we should also update them as well I guess 🙂